src/components/Item/Banner.jsx

```diff
@@ -0,0 +1,51 @@
+import { Link, NavLink } from "react-router-dom";
+
+import logo from "/logo.svg";
+import logo_title from "/logo_title.svg";
+
+import "./Banner.css";
+
+function getLinkStyle({ isActive }) {
+  return {
+    color: isActive ? "var(--blue)" : undefined,
+  };
+}
+
+function Banner() {
+  return (
+    <header className="item-banner">
+      <div className="item-banner-content">
+        <div className="item-banner-left">
+          <div className="item-logo-content">
+            <Link className="item-logo" to="/">
+              <img src={logo} alt="판다마켓 로고 이미지" />
+            </Link>
+            <Link className="item-logo-text" to="/">
+              <img src={logo_title} alt="판다마켓" />
+            </Link>
```

Comment:
💊 제안
아래가 더 적절한 구분일 것 같아요~

```suggestion
            <Link  to="/">
              <img className="item-logo" src={logo} alt="판다마켓 로고 이미지" />
              <img className="item-logo-text" src={logo_title} alt="판다마켓" />
            </Link>
```